apps/web/src/modules/job/apply/components/SelectResume/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ResumeDto } from '@modela/dtos'
+import Box from '@mui/material/Box'
+import FormControl from '@mui/material/FormControl'
+import MenuItem from '@mui/material/MenuItem'
+import TextField from 'common/components/TextField'
+import * as React from 'react'
+
+import { ApplyProps } from './types'
+
+export default function SelectResume(props: ApplyProps) {
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
```

Comment:
you can use `const {id, setId, resumes} = props` 
then use `props.setId(Number(event.target.value))` `value = {id}` and `resumes.map`
for cleaner code